test/matchers/diff.go

```diff
@@ -51,10 +51,10 @@ func (m *lineMatcher) NegatedFailureMessage(actual interface{}) (message string)
 	return ("Expected differences but none found.")
 }
 
-func (m *lineMatcher) normalize(in string) []string {
+func normalize(in string, trim bool) []string {
 	out := []string{}
 	for _, line := range strings.Split(in, "\n") {
-		if m.trim {
+		if trim {
```

Comment:
Nothing to do with this PR, but I'm not quite seeing why we strings.TrimSpace(line) twice.